.github/workflows/release.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           persist-credentials: false
       - name: Install Rust
         run: rustup toolchain add ${{ matrix.rust || 'nightly' }} --no-self-update && rustup default ${{ matrix.rust || 'nightly' }}
-      - uses: taiki-e/install-action@cross
```

Comment:
Changing non-v* branch is wrong. I'm going to revert this and disable dependabot for GHA again.